testing/conftest.py

```diff
@@ -22,7 +20,9 @@ def product_version(self):
 
 @pytest.fixture(scope='session')
 def selenium(request):
-    driver = webdriver.PhantomJS()
+    options = webdriver.FirefoxOptions()
+    options.add_argument('-headless')
```

Comment:
**OPTIONAL**
Selenium throws a warning here, this line doesn't:
```python
options.headless = True
```

Reply:
Yeh I htink that's a nice addition

Reply:
Good call @quarckster looks like its a preferred/deprecated use.